### PR TITLE
[Workers] Add Wrangler Chrome DevTools docs

### PR DESCRIPTION
**Pull request by mrbbot**
Hey! 👋 This change adds docs for https://github.com/cloudflare/workers-sdk/pull/3774, and Wrangler's DevTools integration in general.

### CODE REVIEW

**Review by RamIdeas — APPROVED** [review]

File: content/workers/observability/debug-workers.md [hunk 1]
```diff
@@ -44,6 +44,16 @@ Unlike `wrangler dev`, which provides local instances of resources to develop ag
 
 {{</Aside>}}
 
+### Using DevTools
```
Comment (deadlypants1973) [inline content/workers/observability/debug-workers.md]:
```suggestion
### DevTools
```

Comment (deadlypants1973) [reply content/workers/observability/debug-workers.md]: small fix for no gerunds in titles as per style guide

**Review by deadlypants1973 — APPROVED** [review]
PCX

**Comment by irvinebroque** [comment]
Seems worth a changelog entry! https://developers.cloudflare.com/workers/platform/changelog/

**Pull request merged**